miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultPoolService.java

```diff
@@ -298,7 +298,8 @@ public void validateIndices(Pool pool, Pool beforeChange, Collection<ValidationE
     Set<String> bcIndices = getAllBadIndices(beforeChange);
 
     if(indices.size() > bcIndices.size()){ // If this change introduces new conflicts
-      String errorMessage = "Pools may not contain Library Aliquots with indices with 2 or fewer positions of difference, please address the following conflicts: ";
+      String errorMessage = "Pools may not contain Library Aliquots with indices with 2 or fewer positions of " +
```

Comment:
Since the near-duplicate indices level is now configurable, can `2` be changed to `indexChecker.getWarningMismatches()`?

Reply:
Done, and updated release notes